test/Interop/SwiftToCxx/enums/enum-cxx-clike-enum-case-functions-execution.cpp

```diff
@@ -0,0 +1,181 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend %S/enum-cxx-clike-enum-case-functions.swift -typecheck -module-name Enums -clang-header-expose-public-decls -emit-clang-header-path %t/enums.h
+
+// RUN: %target-interop-build-clangxx -c %s -I %t -o %t/swift-enums-execution.o
+// RUN: %target-interop-build-swift %S/enum-cxx-clike-enum-case-functions.swift -o %t/swift-enums-execution -Xlinker %t/swift-enums-execution.o -module-name Enums -Xfrontend -entry-point-function-name -Xfrontend swiftMain
+
+// RUN: %target-codesign %t/swift-enums-execution
+// RUN: %target-run %t/swift-enums-execution
+
+// REQUIRES: executable_test
```

Comment:
Please also disable it on arm64e : ‘// UNSUPPORTED: CPU=arm64e’

Reply:
Oh yeah I forgot this.